.circleci/config.yml

```diff
@@ -19,9 +19,10 @@ jobs:
     description: Validate formatting of code and documentation
     docker:
       - image: openquantumsafe/ci-ubuntu-bionic-x86_64:latest
-        auth:
-          username: $DOCKER_LOGIN
-          password: $DOCKER_PASSWORD
+# Re-enable iff docker enforces rate limitations without auth:
```

Comment:
This will be happening soon though, right? https://www.docker.com/blog/docker-hub-image-retention-policy-delayed-and-subscription-updates/ says "mid 2021".

https://docs.docker.com/docker-hub/download-rate-limit/

Reply:
Not necessarily: See link at https://github.com/open-quantum-safe/liboqs/issues/884#issuecomment-769158300

Reply:
Ah I see :+1: